### PR TITLE
To string

### PR DESCRIPTION
**Pull request by jbrennan**
Adds an override of `toString()` to all bridged types descending from `NSObject`. That's pretty much all the ones we'd care to log.

### CODE REVIEW

File: PrototopeJSBridge/Context.swift [hunk 1]
```diff
@@ -44,47 +44,64 @@ public class Context {
...
+			
+			if let bridgedNSObject: AnyObject = bridgedType as? NSObject.Type {
+				
+				let bridgedConstructor = JSValue(object: bridgedNSObject, inContext: context)
+				let prototype = bridgedConstructor.valueForProperty("prototype")
+				let toStringImpl: @objc_block () -> String = {
+					let this = JSContext.currentThis()
+					return this.description 
+				}
+				prototype.setFunctionForKey("toString", fn: toStringImpl)
```
Comment (NachoSoto) [inline PrototopeJSBridge/Context.swift]: Nice

Comment (jbrennan) [inline PrototopeJSBridge/Context.swift]: :tophat: tip to @andymatuschak for that one.

File: PrototopeJSBridge/Context.swift [hunk 2]
```diff
@@ -44,47 +44,64 @@ public class Context {
...
-		TextAlignmentBridge.addToContext(context)
-		CameraLayerBridge.addToContext(context)
-		CameraPositionBridge.addToContext(context)
-        ShapeLayerBridge.addToContext(context)
-        SegmentBridge.addToContext(context)
-        LineCapStyleBridge.addToContext(context)
-        LineJoinStyleBridge.addToContext(context)
+		
+		
+		// Capturing `context` here (instead of passing it in as a parameter) causes the compiler to segfault as of Swift 1.2b4.
```
Comment (andymatuschak) [inline PrototopeJSBridge/Context.swift]: Yeah, you have to use a closure instead of an inner function. :(

**Comment by andymatuschak** [comment]
Frickin' sweet!

**Pull request merged**